app/src/main/java/com/eveningoutpost/dexdrip/Models/BgReading.java

```diff
@@ -240,16 +246,10 @@ public static void create(EGVRecord egvRecord, long addativeOffset, Context cont
                 bgReading.raw_calculated = (((calSlope * bgReading.raw_data) + calIntercept) - 5);
             }
             Log.i(TAG, "create: NEW VALUE CALCULATED AT: " + bgReading.calculated_value);
-            bgReading.calculated_value_slope = bgReading.slopefromName(egvRecord.getTrend().friendlyTrendName());
+            AtomicBoolean hide = new AtomicBoolean();
+            bgReading.calculated_value_slope = bgReading.slopefromName(egvRecord.getTrend().friendlyTrendName(), hide);
```

Comment:
'hide' looks like a parameter passed to the function but it actually is a return parameter. Is this a wanted side-effect?
I'd prefer it as real return value if is a return value. (class bundling a `double` and a `boolean`).